docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1965,6 +1965,18 @@ Enter a comma-separated list of plugin identifiers to hide in the plugin catalog
 
 <hr>
 
+## [plugin.plugin_id]
```

Comment:
maybe it's better to move this before `## [plugin.grafana-image-renderer]` (line 2023) to have all the [plugin.*] related sections close together